allensdk/brain_observatory/ecephys/ecephys_session.py

```diff
@@ -579,8 +595,8 @@ def _build_units_table(self, units_table):
         table.index.name = 'unit_id'
         table = table.rename(columns={
             'description': 'probe_description',
-            'manual_structure_id': 'structure_id',
-            'manual_structure_acronym': 'structure_acronym',
+            #'manual_structure_id': 'structure_id',
+            #'manual_structure_acronym': 'structure_acronym',
```

Comment:
Not quite sure why you are doing this. In any case, if we don't want to name columns this way, we should remove the commented lines!